en/power_module/can_pmu.md

```diff
@@ -0,0 +1,79 @@
+# CAN PMU
+
+![CAN PMU](../../assets/hardware/power_module/can_pmu.jpg)
```

Comment:
@davids5 When you have a chance, can you please sanity check the specific pages here for the power modules CUAV added? There is one that uses CAN and another that is analog voltage/current.

Reply:
@hamishwillee - I do not have the HW, now have reviewed the designs, so I am not sure what value I can add here.

Reply:
@davids5 Fair point. Would it be useful for you to have? We could request?